app/assets/javascripts/user.service.js.erb

```diff
@@ -0,0 +1,33 @@
+(function(){
+
+'use strict';
+
+angular.module('GeoBox')
+.service('UserService', UserService);
+
+UserService.$inject = ['$http', '$rootScope'];
+function UserService($http, $rootScope){
+	var user_service = this;
+	user_service.user = {};
+
+	user_service.baseUrl = "<% if Rails.env.production? %>			\
+								<%= 'https://geobox-staging.herokuapp.com' %> 	\
+						  	<% else %>								\
+						  		<%= 'http://localhost:3000' %> 			\
+						  	<% end %>".trim();
+	user_service.getUserInfo = function(){
+		return $http({
+			method: 'POST',
+			url: user_service.baseUrl + '/user_info'
+		})
+		.then(function(response){
+			user_service.user = response.data;
```

Comment:
The user controller is POSTed but i am wondering what the response.data looks like?

Reply:
I see it in user controller get userinfo.

Reply:
hm, perhaps it would make more sense if this was a get request

Reply:
i've changed it to a get request